api/search.rb

```diff
@@ -64,15 +82,19 @@ def get_thread_ids(context, group_ids, local_params, search_text)
 
 def get_suggested_text(search_text)
   body = {
+    suggest: {
       suggestions: {
           text: search_text,
           phrase: {
-              field: :_all
+              field: :body
           }
       }
+    }
   }
-  response = Elasticsearch::Model.client.suggest(index: Content::ES_INDEX_NAME, body: body)
-  suggestions = response.fetch('suggestions', [])
+
+  response = Elasticsearch::Model.client.search(index: TaskHelpers::ElasticsearchHelper::INDEX_NAMES, body: body)
+  suggestions = response["suggest"].fetch('suggestions', [])
```

Comment:
Please keep your code clean. Do not mix up quotes.

Reply:
Please check it everywhere in your new code.

Reply:
Done

Reply:
Thanks, here I use single quates